tick/array/tests/serialize_test.py

```diff
@@ -60,6 +60,33 @@ def test_serialize_sparse_2d_array(self):
         # python 3.5 has show to required this - investigate typemappers
         gc.collect()
 
+    def test_serialize_column_major_2d_array(self):
+        """...Test serialization of 2d dense array is done as expected
+        """
+        x = np.array([[1, 2.1, 5], [0, 4, 1]]).astype(self.dtype)
+        array = np.asfortranarray(x).reshape(3, 2)
```

Comment:
Don't reshape it, we want it to be the column major representation of a 2 lines, 3 columns array